std/format.d

```diff
@@ -2013,8 +2013,33 @@ private void formatUnsigned(Writer, T, Char)
     if (prefix1) put(w, prefix1);
     if (prefix2) put(w, prefix2);
 
-    foreach (i ; 0 .. zerofill)
-        put(w, '0');
+    if (fs.flSeparator)
+    {
+        if (zerofill > 0)
+        {
+            put(w, '0');
+            --zerofill;
+        }
+
+        int j = fs.width;
```

Comment:
should this be uint?

Reply:
FormatSpec.width is an int. So I think int is good